pnpm-lock.yaml

```diff
@@ -136,6 +136,7 @@ importers:
       acorn: ^8.8.1
       caniuse-lite: ^1.0.30001451
       cheerio: 1.0.0-rc.12
+      fork-ts-checker-webpack-plugin: 8.0.0
```

Comment:
1000+ lines added to lock file, we can use `pnpm dedupe` to remove duplicated dependencies.